tests/IntegrationTest.php

```diff
@@ -56,26 +56,14 @@ protected static function assertRevertedOrderFixing(IntegrationCase $case, strin
         }
 
         if ($settings['isExplicitPriorityCheck']) {
-            if ($fixedInputCode === $fixedInputCodeWithReversedFixers) {
-                if (\in_array($case->getFileName(), [
-                    'priority'.\DIRECTORY_SEPARATOR.'backtick_to_shell_exec,escape_implicit_backslashes.test',
-                    'priority'.\DIRECTORY_SEPARATOR.'backtick_to_shell_exec,string_implicit_backslashes.test',
-                    'priority'.\DIRECTORY_SEPARATOR.'no_unused_imports,blank_line_after_namespace_2.test',
-                    'priority'.\DIRECTORY_SEPARATOR.'single_import_per_statement,no_unused_imports.test',
-                ], true)) {
-                    self::markTestIncomplete(\sprintf(
-                        'Integration test `%s` was defined as explicit priority test, but no priority conflict was detected.'
-                        ."\n".'Either integration test needs to be extended or moved from `priority` to `misc`.'
-                        ."\n".'But don\'t do it blindly - it deserves investigation!',
-                        $case->getFileName()
-                    ));
-                }
-            }
-
             self::assertNotSame(
                 $fixedInputCode,
                 $fixedInputCodeWithReversedFixers,
-                \sprintf('Test "%s" in "%s" is expected to be priority check.', $case->getTitle(), $case->getFileName())
+                \sprintf(
+                    'Test "%s" in "%s" is expected to be priority check, but fixers applied in revered order made the same changes.',
```

Comment:
```suggestion
                    'Test "%s" in "%s" is expected to be priority check, but fixers applied in reversed order made the same changes.',
```

Reply:
I hate when type is another correct word...